src/System.Private.CoreLib/src/System/DateTime.Win32.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Diagnostics.Contracts;
+
+namespace System
+{
+    public partial struct DateTime : IComparable, IFormattable, IComparable<DateTime>, IEquatable<DateTime>, IConvertible
+    {
+        public static DateTime UtcNow
+        {
+            [System.Security.SecuritySafeCritical]  // auto-generated
+            get
+            {
+                Contract.Ensures(Contract.Result<DateTime>().Kind == DateTimeKind.Utc);
+                // following code is tuned for speed. Don't change it without running benchmark.
```

Comment:
Could please delete this comment, and instead add the following comment in front of `new Date`: `For performance, use a private constructor that does not validate arguments`